src/machine/board_nano-33-ble.go

```diff
@@ -72,8 +83,19 @@ const (
 
 // I2C pins
 const (
-	SDA_PIN = P0_31
-	SCL_PIN = P0_02
+	// Defaults to internal
+	SDA_PIN = SDA1_PIN
+	SCL_PIN = SCL1_PIN
+
+	// I2C0 (external) pins
+	SDA0_PIN = P0_31
+	SCL0_PIN = P0_02
+
+	// I2C1 (internal) pins
+	SDA1_PIN = P0_14
+	SCL1_PIN = P0_15
+
+	I2C_PULLUP = P1_00 // Set high for I2C to work
```

Comment:
This is odd, but required for internal I2C to work.
Extracted from https://github.com/arduino/ArduinoCore-mbed/blob/master/variants/ARDUINO_NANO33BLE/variant.cpp#L159